src/components/credit-card-expire/code/views/creditCardExpireView.jsx

```diff
@@ -0,0 +1,74 @@
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    label: React.PropTypes.string,
+    expireDate: React.PropTypes.string,
+    defaultValue: React.PropTypes.string,
+    name: React.PropTypes.string,
+    id: React.PropTypes.string
+  },
+
+  getInitialState: function() {
+    var expireDate = this.updateValue(this.props.defaultValue);
+    return {
+      expireDate: expireDate || ''
+    };
+  },
+
+  getDefaultProps: function() {
+    return {
+      formatInput: false,
+      placeHolder: 'MM/YY',
+      name: 'credit-card-expire',
+      id: 'credit-card-expire'
+    };
+  },
+
+  updateValue: function(expireDate) {
+
+    if(typeof expireDate === 'undefined')
```

Comment:
Any chance we could change it so that the opening curly brace is on the same line as the if?